### DISCUSSION TITLE
Refresh after 5-10 sec

### DISCUSSION THREAD

**Discussion by Kolionixee** [maximmax42/Discord-CustomRP#252] [Ideas]
Refresh after 5-10 sec because when you start playing like gta v it changes it to gta v not Custom rpc.

**Comment by maximmax42** [comment]
Not really viable because presence needs to be changed every time you update it. Plus, I think you need to reconnect every time you launch a game with presence.

**Reply by Kolionixee** [reply]
Oh ok :(
